src/ftw_cli/cli.py

```diff
@@ -106,13 +108,14 @@ def inference_run(input, model, out, resize_factor, gpu, patch_size, batch_size,
 
 @inference.command("polygonize", help="Polygonize the output from inference for the raster image given via INPUT.")
 @click.argument('input', type=click.Path(exists=True), required=True)
-@click.option('--out', '-o', type=str, required=True, help="Output filename for the polygonized data.")
+@click.option('--out', '-o', type=str, required=True, help="Output filename for the polygonized data. " + SUPPORTED_POLY_FORMATS_TXT)
 @click.option('--simplify', type=float, default=None, help="Simplification factor to use when polygonizing.")
 @click.option('--min_size', type=float, default=500, help="Minimum area size in square meters to include in the output.")
```

Comment:
```suggestion
@click.option('--min_size', type=float, default=500, help="Minimum area size in square meters to include in the output (default is 500).")
```

Reply:
The better thing to do is:

```suggestion
@click.option('--min_size', type=float, default=500, show_default=True, help="Minimum area size in square meters to include in the output.")
```

Reply:
Ah, perfect - as I wrote that I felt there was likely a better way but didn't see other examples right there.